### PR TITLE
Bump up ks-devops helm charts

### PR DESCRIPTION
**Pull request by JohnNiang**
### What this PR dose

1. Bump up ks-devops helm charts to 0.1.9.
2. Add missing docker images of devops agents.

### Why we need it

For next release(v3.2.1).

/kind chore
/area devops
/cc @kubesphere/sig-devops

### CODE REVIEW

File: roles/download/defaults/main.yml [hunk 1]
```diff
@@ -403,6 +411,12 @@ images:
     sha256: "{{ builder_maven_checksum|default(None) }}"
     group: "kubesphere-devops-images"
 
+  builder_python:
```
Comment (LinuxSuRen) [inline roles/download/defaults/main.yml:414]: I prefer to split this PR into two. We don't have python on v3.2.0. And parts of the new changes should be merged into the release-3.0

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
/cherrypick release-3.2

File: roles/download/defaults/main.yml (hunk 1 above)
Comment (JohnNiang) [reply roles/download/defaults/main.yml:414]: I have searched builder-python docker image from DockerHub, please see 
- https://hub.docker.com/layers/kubespheredev/builder-python/v3.2.0/images/sha256-db78f59efc04f6feb1dcdb00eed4c6b4f006b0e7f43b6cadcf16541159e16b56?context=explore
- https://hub.docker.com/layers/kubespheredev/builder-python/v3.2.0-podman/images/sha256-7bdf1355ae88e5cf526a484a5d73247d302555b3623bfcafc7f0d73775fc7a86?context=explore

File: roles/download/defaults/main.yml [hunk 2]
```diff
@@ -403,6 +411,12 @@ images:
...
+    repo: "{{ builder_registry }}/builder-maven"
```
Comment (LinuxSuRen) [inline roles/download/defaults/main.yml:415]: maven or python?

File: roles/download/defaults/main.yml [hunk 3]
```diff
@@ -415,6 +429,42 @@ images:
...
+    group: "kubesphere-devops-images"
+
+  builder_maven_podman:
+    repo: "{{ builder_registry }}/builder-maven"
+    tag: "{{ builder_maven_podman_tag }}"
+    sha256: "{{ builder_maven_podman_checksum|default(None) }}"
+    group: "kubesphere-devops-images"
+
+  builder_python_podman:
+    repo: "{{ builder_registry }}/builder-maven"
```
Comment (LinuxSuRen) [inline roles/download/defaults/main.yml:451]: python or maven?

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]

**Review by LinuxSuRen — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**